package.json

```diff
@@ -66,25 +42,49 @@
     "@types/remove-markdown": "^0.3.1",
     "@typescript-eslint/eslint-plugin": "^6.2.0",
     "@typescript-eslint/parser": "^6.2.0",
+    "@uma/common": "^2.34.0",
+    "@uma/contracts-frontend": "^0.4.17",
     "@uma/contracts-node": "^0.4.17",
+    "@uma/sdk": "^0.34.2",
     "autoprefixer": "^10.4.14",
     "chromatic": "^6.20.0",
+    "date-fns": "^2.30.0",
     "eslint": "8.45.0",
     "eslint-config-next": "13.4.12",
     "eslint-config-prettier": "^8.9.0",
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-storybook": "^0.6.13",
+    "ethers": "^5.7.2",
+    "framer-motion": "^10.14.0",
+    "fuse.js": "^6.6.2",
+    "graphql-request": "^6.1.0",
+    "immer": "^10.0.2",
+    "lodash": "^4.17.21",
     "msw": "^1.2.3",
     "msw-storybook-addon": "^1.8.0",
+    "next": "13.4.12",
     "next-plugin-svgr": "^1.1.8",
     "postcss": "^8.4.27",
     "prettier": "^3.0.0",
     "prettier-plugin-tailwindcss": "^0.4.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-focus-on": "^3.9.1",
+    "react-loader-spinner": "^5.3.4",
+    "react-loading-skeleton": "^3.3.1",
+    "react-markdown": "^8.0.7",
+    "remove-markdown": "^0.5.0",
     "storybook": "^7.1.1",
     "storybook-mock-date-decorator": "^1.0.1",
+    "styled-components": "^6.0.5",
+    "superstruct": "^1.0.3",
+    "swr": "^2.2.0",
     "tailwindcss": "^3.3.3",
     "ts-node": "^10.9.1",
-    "typescript": "5.1.6"
+    "typescript": "5.1.6",
+    "use-immer": "^0.9.0",
+    "usehooks-ts": "^2.9.1",
+    "wagmi": "^1.4.13"
```

Comment:
this is the first version where they added core to list of chains ( looked up the blame )